src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, value) {
+      callback(value);
```

Comment:
You should bind your callback function to the context via the `call` method for access to `this`, because now your callback function knows nothing about the object